clearwater-nginx/usr/share/clearwater-nginx/nginx.monit

```diff
@@ -39,21 +39,32 @@ check process nginx_process pidfile /var/run/nginx.pid
 
   # Kill all running nginx processes before starting up (as if any slave processes are still running after the master process
   # dies then nginx can't restart
-  start program = "/bin/bash -c 'killall -9 nginx; /usr/share/clearwater/bin/issue-alarm monit 5001.3; /etc/init.d/nginx start'"
-  stop program = "/bin/bash -c '/usr/share/clearwater/bin/issue-alarm monit 5001.3; /etc/init.d/nginx stop'"
-  restart program = "/bin/bash -c 'killall -9 nginx; /usr/share/clearwater/bin/issue-alarm monit 5001.3; /etc/init.d/nginx restart'"
+  start program = "/bin/bash -c 'killall -9 nginx;
+                   /usr/share/clearwater/bin/issue-alarm monit 5001.3;
```

Comment:
Can you pull this into a run-in-nginx-namespace script? (e.g. https://github.com/Metaswitch/clearwater-infrastructure/blob/master/clearwater-infrastructure/usr/share/clearwater/bin/run-in-signaling-namespace)